yunikorn-web.yaml

```diff
@@ -0,0 +1,81 @@
+package:
+  name: yunikorn-web
+  version: 1.6.0
+  epoch: 0
+  description: Apache YuniKorn Web UI
+  copyright:
+    - license: Apache-2.0
+
+environment:
+  contents:
+    packages:
+      - node-gyp
+      - nodejs-18
```

Comment:
Does this need to be 18 or can it be `nodejs`?

Reply:
it gave a warning like nodejs 18 is needed something like that, this is why I added this.